MonoGame.Framework.Content.Pipeline/TargetPlatform.cs

```diff
@@ -85,7 +85,17 @@ public enum TargetPlatform
         /// <summary>
         /// WebAssembly and Bridge.NET
         /// </summary>
-        Web
+        Web,
+
+        /// <summary>
+        /// Windows GDK
+        /// </summary>
+        DesktopGDK,
```

Comment:
Sorry @Togimaro  i thought i had commented on this a few days ago, but couldn't find it.

The term "Desktop" here is used to indicate that platform works on all desktop PCs... 
 Windows, Mac, and Linux.  This is why we have `DesktopGL`.

`DesktopGDK` would then imply GDK works on Mac and Linux... which it does not.

So `WindowsGDK` would be the right name here as it really is Windows desktop only.

It also then matches the other platform enum `MonoGamePlatform`.

Reply:
Fixed!